parserasgeo/features/cross_section.py

```
@@ -514,6 +520,76 @@ def cut_line_ratio(self):
+        If they are not defined at the bank stations, add Manning's n values at those points.
+        """
+        
+        left_bank = self.bank_sta.left
+        right_bank = self.bank_sta.right
+        banks = [left_bank, right_bank]
+        
+        manning_tuples = self.mannings_n.values
+        manning_stations = [x[0] for x in manning_tuples]
+        
+        for bank in banks:
+            # if the bank station is not in the list, grab the Manning's n to the left
+            if bank not in manning_stations:
+                left_manning = [x[1] for x in manning_tuples if x[0] < bank]
+                left_index = len(left_manning)
+                left_manning = left_manning[-1]
+                manning_insert = (bank, left_manning, 0)
+                manning_tuples.insert(left_index, manning_insert)
+        
+        # do not pull the n-value at the right bank because that defines the n-values to the right of the bank
+        self.channel_n = [x for x in manning_tuples if (x[0] >= banks[0] and x[0] < banks[1])]        
+            
+    def alter_channel_n(self, scalar):
+        """
+        Alters the channel n-values by a scaling factor
+        """
+        
+        if self.channel_n is not None:
+            new_channel_n = []
+            for n in self.channel_n:
+                temp_n = n[1]*scalar
+                temp_tuple = (n[0], temp_n, 0)
+                new_channel_n.append(temp_tuple)
+            
+            ind = 0
+            for old_n in self.mannings_n.values:
+                for new_n in new_channel_n:
+                    if old_n[0] == new_n[0]:
+                        self.mannings_n.values.pop(ind)
+                        self.mannings_n.values.insert(ind, new_n)
+                        break
+                ind += 1
+                
+            self.channel_n = new_channel_n
+        else:
+            pass
```

It seems like this should raise an exception instead of failing silently.